modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/config/types.scala

```diff
@@ -95,7 +114,9 @@ object types {
       OneTimeReward(EpochProgress(1928500L), treasureWalletMetanomics1, TransactionAmount(150_000_000_000_000_00L)),
       OneTimeReward(EpochProgress(1928500L), treasureWalletMetanomics2, TransactionAmount(150_000_000_000_000_00L)),
       OneTimeReward(EpochProgress(1928500L), treasureWalletMetanomics3, TransactionAmount(150_000_000_000_000_00L))
-    )
+    ),
+    // Fallback reward pool in case no emission formula is configured
+    fallbackRewardPoolPerEpoch: Amount = Amount(10000000000L)
```

Comment:
Does this ever actually get triggered? Is this dev only?

Reply:
this was removed, there is a fallback in case no emission config is found and the epoch is not covered by the provided `rewardsPerEpoch`. If that val is correct, we should not hit this case.